Always use assembly over mutex since the mutex version fails to build
a shared library. See #15045 for details.

diff --git a/tune/threads/tune_count.c b/tune/threads/tune_count.c
index f09717f..4dc3fde 100644
--- a/tune/threads/tune_count.c
+++ b/tune/threads/tune_count.c
@@ -241,8 +241,8 @@ int main(int nargs, char **args)
  */
       if (tmut < tldec*1.02)
       {
-         printf("\nNO REAL ADVANTAGE TO ASSEMBLY, FORCING USE OF MUTEX\n");
-         ATL_assert(!system("make iForceUseMutex"));
+         printf("\nNO REAL ADVANTAGE TO ASSEMBLY OVER MUTEX\n");
+         printf("\nASSEMBLY/MUTEX ratio is %.2f, but we'll stick with assembly anyway\n", tldec/tmut);
       }
    }
    free(timearr);
